client/config/config.go

```diff
@@ -29,6 +31,11 @@ func SetConfigs() {
 	RepositoryURL = os.Getenv(`HUSKYCI_CLIENT_REPO_URL`)
 	RepositoryBranch = os.Getenv(`HUSKYCI_CLIENT_REPO_BRANCH`)
 	HuskyAPI = os.Getenv(`HUSKYCI_CLIENT_API_ADDR`)
+	languagesToExclude := strings.Split(os.Getenv(`HUSKYCI_LANGUAGE_EXCLUSIONS`), ",")
```

Comment:
It would be good if we have this environment variable updated here: https://huskyci.opensource.globo.com/docs/api/env-vars What is the default value, for instance?

Reply:
Will open a PR to add to the documentation!